tests/plugins/divarea/manual/detached.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: 4.17.0, bug, 4462, divarea
+@bender-ui: collapsed
+@bender-ckeditor-plugins: divarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+
+1. Open dev console
+
+2. Click "Toggle" button twice. To hide and show editor.
+
+	**Expected:** Editor shows with same data. Content is editable. Elementspath (at the bottom is updated).
+
+	**Unexpected:** Editor data is lost, content area is not editable.
+
+3. Write some text in editor.
+
+4. Repeat step 2.
+
+	**Expected:** Editor shows with same data. Content is editable. Elementspath (at the bottom is updated).
```

Comment:
If you follow the test carefully, Elementspath will be blank at this point - it will be updated once you focus an editor. I wonder - was there any issue with this plugin during development? Or is there another reason why you wanted to include this check?

Reply:
I've changed the test description to be more precise.  
  
This plugin was my first indicator that something is not working. At the very beginning when I restore the data - it appears properly in the content area. However, the bottom bar has remained blank. Even if the content was editable.

That directed me to use build-in mechanics and flows as in the original editor creation flow. There is a lot of bindings to many events. So it will be hard to manually recreate all of them. It was better to somehow repeat all bindings automatically.  
  
So editable content and working `elementspath` plugin were my mine manual indicators for this solution. Then unit tests (mostly jQuery adapters) indicate if my working solution remains stable with other parts of the editor.